universal-application-tool-0.0.1/app/views/components/FieldWithLabel.java

```diff
@@ -90,15 +97,16 @@ public ContainerTag getContainer() {
     } else {
       fieldTag.withValue(this.fieldValue);
     }
-    fieldTag.withId(this.id).withName(id).withPlaceholder(this.placeholderText);
 
-    if (formId.length() > 0) {
-      fieldTag.attr("form", formId);
-    }
+    fieldTag
+        .withCondId(!Strings.isNullOrEmpty(this.id), this.id)
```

Comment:
praise: Oh! This is great. We can use this `withCond` pattern here for `value` if/when we want to be able to pre-set a value in this component. We'll want that for applicant view inputs.